qp_klp/klp.py

```diff
@@ -246,6 +246,12 @@ def sequence_processing_pipeline(qclient, job_id, parameters, out_dir):
                 csv_fps.append(join(root, csv_file))
 
         for project, upload_dir in special_map:
+            if sifs:
```

Comment:
Note that this will fail if a project has no blanks but other do; you need to check per specific project.

Reply:
Not pretty but this works:
``` if sifs and [sif for sif in sifs if f'{project}_blanks.tsv' in sif]```